fe/lib/components/LocationSuggest/LocationSuggest.tsx

```diff
@@ -182,7 +186,7 @@ export const LocationSuggest = forwardRef<
           <input
             type="hidden"
             name={name}
-            value={selectedLocationId}
+            value={selectedLocation?.id.value}
```

Comment:
Providing an `undefined` value to a form element is not desirable as it toggles the component between controlled and uncontrolled.

Reply:
Fixed